app/views/questions/show.html.erb

```diff
@@ -1 +1,25 @@
-<%= render "question" %>
\ No newline at end of file
+<h2><%= @question.title %></h2>
+<%= @question.content %>
+<%= link_to "Edit", edit_question_path %>
+<%= link_to "Delete", question_path, method: :delete %>
+<div class="row">
+  <div class="col-md-10 col-md-offset-1">
+    <h5>Comments</h5>
+    <% @question.comments.each do |comment| %>
```

Comment:
We should do some better structured HTML around this.

Also:  `render @question.comments` ?